<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuxesDialog</class>
 <widget class="QDialog" name="AuxesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Вибрационный источник</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt; Номер трассы АКФ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>akfTraceNbSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="akfTraceNbSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt; Минимальная частота, Гц&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>minFreqSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="minFreqSpinBox">
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt; Максимальная частота, Гц&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>maxFreqSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="maxFreqSpinBox">
           <property name="minimum">
            <number>50</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Время максимальной амплитуды, мс</string>
           </property>
           <property name="buddy">
            <cstring>maxTimeSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="maxTimeSpinBox">
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Уровень частот свип-сигнала, дБ</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="freqLevelSpinBox">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Амплитуда АКФ (1 накопление)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="akfAmplDoubleSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>99999999999999996863366107917975552.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="openSegdPushButton">
         <property name="text">
          <string>Открыть образец</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>153</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt; - Заполняется пользователем&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Взрывной источник</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="timeBreakGroupBox">
         <property name="title">
          <string>Отметка момента</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QCheckBox" name="timeBreakCheckBox">
            <property name="text">
             <string>Проверять</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="timeBreakSettingsGroupBox">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="timeBreakTraceNbLabel">
               <property name="text">
                <string>Номер трассы</string>
               </property>
               <property name="buddy">
                <cstring>timeBreakTraceNbSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="timeBreakTraceNbSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="timeBreakTmaxLabel">
               <property name="text">
                <string>Tmax</string>
               </property>
               <property name="buddy">
                <cstring>timeBreakTmaxSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="timeBreakTmaxSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="timeBreakAmaxLabel">
               <property name="text">
                <string>Amax</string>
               </property>
               <property name="buddy">
                <cstring>timeBreakAmaxLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="timeBreakAmaxLineEdit">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="timeBreakNbOfDiscretLabel">
               <property name="text">
                <string>Количество дискретов</string>
               </property>
               <property name="buddy">
                <cstring>timeBreakNbOfDiscretSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="timeBreakNbOfDiscretSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="upholeTimeGroupBox">
           <property name="title">
            <string>Аналоговое Тв</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="upholeTimeCheckBox">
              <property name="text">
               <string>Проверять</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="upholeTimeTraceNbLabel">
              <property name="text">
               <string>Номер трассы</string>
              </property>
              <property name="buddy">
               <cstring>upholeTimeTraceNbSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="upholeTimeTraceNbSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="confirmedTimeBreakGroupBox">
         <property name="title">
          <string> Подтвержденная отметка момента</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QCheckBox" name="confirmedTimeBreakCheckBox">
            <property name="text">
             <string>Проверять</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="confirmedTimeBreakSettingsGroupBox">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="confirmedTimeBreakTraceNbLabel">
               <property name="text">
                <string>Номер трассы</string>
               </property>
               <property name="buddy">
                <cstring>confirmedTimeBreakTraceNbSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="confirmedTimeBreakTraceNbSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="confirmedTimeBreakTmaxLabel">
               <property name="text">
                <string>Tmax</string>
               </property>
               <property name="buddy">
                <cstring>confirmedTimeBreakTmaxSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="confirmedTimeBreakTmaxSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="confirmedTimeBreakAmaxLabel">
               <property name="text">
                <string>Amax</string>
               </property>
               <property name="buddy">
                <cstring>confirmedTimeBreakAmaxLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="confirmedTimeBreakAmaxLineEdit">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="confirmedTimeBreakNbOfDiscretLabel">
               <property name="text">
                <string>Количество дискретов</string>
               </property>
               <property name="buddy">
                <cstring>confirmedTimeBreakNbOfDiscretSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="confirmedTimeBreakNbOfDiscretSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openExamplePushButton">
           <property name="text">
            <string>Открыть образец</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuxesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeBreakCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeBreakSettingsGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmedTimeBreakCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmedTimeBreakSettingsGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upholeTimeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>upholeTimeTraceNbSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
